homeUi/src/main/kotlin/com/gravatar/app/homeUi/presentation/home/profile/ProfileHeader.kt

```diff
@@ -0,0 +1,123 @@
+package com.gravatar.app.homeUi.presentation.home.profile
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.gravatar.extensions.defaultProfile
+import com.gravatar.restapi.models.Profile
+import com.gravatar.ui.components.atomic.Avatar
+
+@Composable
+fun ProfileHeader(
+    profile: Profile,
```

Comment:
I'm using the `Profile` class from the Gravatar-SDK. If that were from a third-party library, it would be a good idea to map that object to another one managed by the app. In this case, since we are using our library, I consider it under our control.